src/App.css

```diff
@@ -1,24 +1,72 @@
-.App {
+.main {
+  background-color: #ff5698;
+}
```

Comment:
- [ ] Kindly make sure your background image fits the whole size of the screen in order to match the Figma design as much as possible.

   _**Hint: You can add the `height` property with `100vh` as a value which will give the `main` container ability to fit the whole visible screen**_


   ### YOUR DESIGN
    
   ![image](https://user-images.githubusercontent.com/87197412/233772055-c07497cf-8720-46c0-94c6-a55beccaebda.png)


   ### YOUR DESIGN AFTER ADDING THE HEIGHT
     ![image](https://user-images.githubusercontent.com/87197412/233772010-ba6dad78-13d0-4898-83be-61bc5af0f295.png)


   ### FIGMA DESIGN
    ![image](https://user-images.githubusercontent.com/87197412/233771954-72dd3df5-85f3-4f11-bf78-acc691db0c33.png)